dashboard/app/models/levels/level_group.rb

```diff
@@ -196,4 +196,17 @@ def self.get_survey_results(script, section)
       end
     end.compact
   end
+
+  # @param {User} current_user - The currently signed in user
+  # @param {User} user - The optional user we're trying to see the attempt of
+  # @param {Level} level - The sublevel we'd like the last attempt for
```

Comment:
This might be a dumb question: What's a sublevel?

Reply:
A LevelGroup consists of a number of individual levels (i.e. mix of match, multi, etc.). These are called sublevels.